nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/csv/CSVReader.java

```diff
@@ -60,6 +61,17 @@
     private volatile String timestampFormat;
     private volatile boolean firstLineIsHeader;
     private volatile boolean ignoreHeader;
+    private volatile String charSet;
+
+    public static final PropertyDescriptor INPUT_CHARSET = new PropertyDescriptor.Builder()
```

Comment:
Can we move this property into CSVUtils as the other properties? It seems to be more in line with the initial intent. Besides, you could use a single property for both reader and writer. Just need to have a more generic description like "The Character Encoding that is used to encode/decode the CSV file". What do you think?